src/main/java/org/opentripplanner/model/Trip.java

```diff
@@ -110,6 +115,17 @@ public void setTripShortName(String tripShortName) {
         this.tripShortName = tripShortName;
     }
 
+    /**
+     * Internal code (non-public identifier) for the journey (e.g. train- or trip number from
+     * the planners' tool). This is kept to ensure compatibility with legacy planning systems.
+     * There is no GTFS equivalent.
```

Comment:
In NeTEx this maps to {@link ServiceJourney#privateCode}, there is no GTFS equivalent.